ccw.core/src/clj/ccw/leiningen/generic_launch.clj

```diff
@@ -26,16 +26,16 @@
         args (str/trim args)]
     (launch/lein project-name args)))
 
-(defn generic-launch 
+(defn generic-launch
   "Open a popup asking the user the Leiningen command to issue,
    and then dynamically create a Java launch configuration invoking the command"
   [project]
-  (swt/ui display
-    (let [dialog	 (doto (swt/new-shell display
-                                       SWT/ON_TOP
-                                       ;SWT/TITLE
-                                       ;SWT/CLOSE
-                                       SWT/APPLICATION_MODAL)
+  (swt/ui (let [display (swt/display)
```

Comment:
Thinking about it again, maybe I can make this explicit and pass current already here...given that I am modifying these files...and pass swt/display as default for now, what do you think? Better now or later?